### DISCUSSION TITLE
taiko playfield size is so big

### DISCUSSION THREAD

**Discussion by Sebola3461** [ppy/osu#21899] [Ideas] [answered]
# What's wrong?
Play taiko in lazer is weird cuz every time when i change from stable to lazer i feel the playfield size difference, this new scale is weird cuz it 's so painful for the "reading" skill. This is why i don't play on lazer, i literally can't play well with this playfield sizing 

# Lazer
![osu_2022-12-28_15-36-51](https://user-images.githubusercontent.com/62405013/209857740-628fb1b2-9507-4aaa-aa10-aae31d959f1d.jpg)

# Stable
![image](https://user-images.githubusercontent.com/62405013/209857877-68d6e43c-0ac0-4dff-9da7-deb78c79c81b.png)

# Side by Side
![image](https://user-images.githubusercontent.com/62405013/209858088-fee18208-7612-42a4-99b5-0af1e87f064b.png)

# Client Config
- Stable: 1360x768 fullscreen
- Lazer: 1360x760 fullscreen with default UI scaling

**Comment by Theighlin** [comment]
The playfield aspect ratio changes are intentional. Does this still happen with classic mod enabled?

**Reply by Sebola3461** [reply]
Yes it happen. But this isn't the issue here, i'm talking about the intentional aspect ratio change

**Comment by vegguid** [comment]
If you use the classic skin that is IIRC included, does it then also look different?

**Reply by Sebola3461** [reply]
Yes. (Lazer skin is with more opacity)
![image](https://user-images.githubusercontent.com/62405013/209859453-0da6e6dd-043d-468a-a27c-796901cb2d87.png)

**Comment by bdach** [comment, marked as answer]
duplicate of https://github.com/ppy/osu/discussions/13047
